PrivateClinicsWebNet.Application/DTOs/LoginDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PrivateClinicsWebNet.Application.DTOs;
+
+public record LoginDto
+{
+    public string Email { get; set; }
+    public string Password { get; set; }
+}
```

Comment:
U loose benefits of records bc u use `set` but not init. I recommend u use primary constructors like LoginDto(string Email, string Password)